pkg/ui/workspaces/db-console/src/views/hotRanges/hotRangesTable.tsx

```diff
@@ -44,6 +44,19 @@ interface HotRangesTableProps {
   onSortChange?: (ss: SortSetting) => void;
 }
 
+const tableToCell = (val: any) => {
+  // make an assumption if we see a single table
+  if (val.tables.length === 1) {
+    return (
+      <Link to={util.EncodeDatabaseTableUri(val.databases[0], val.tables[0])}>
```

Comment:
This link looks like it's to the legacy page. For the new pages we use the table id. I guess that might make it difficult to perform this linking since the hot ranges don't seem to return any ids.

Reply:
Gotcha, I'll see if there's a way to link it to the new databases page if possible

Reply:
Checked in, it's okay for the link to be omitted as part of this release.